src/Compilers/Core/Portable/CodeAnalysis.csproj

```diff
@@ -63,6 +63,7 @@
     <PackageReference Include="System.ValueTuple" Version="$(SystemValueTupleVersion)" />
     <PackageReference Include="System.Xml.XPath.XDocument" Version="$(SystemXmlXPathXDocumentVersion)" />
     <PackageReference Include="System.Text.Encoding.CodePages" Version="$(SystemTextEncodingCodePagesVersion)" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsVersion)" />
```

Comment:
Does this need updates to the VisualStudioSetup.Dependencies package? Will it ship a new façade that we need to be ready for in RPS?

Reply:
(VS integration test failures imply yes, this needs more façade updates.)

Reply:
Likely.

Reply:
... and binding redirects.